app/views/submissions/new.html.erb

```diff
@@ -0,0 +1,41 @@
+<div class="show_and_edit">
+  <%= form_for new_post do |post| %>
+    <% if new_post.errors[:base].present? %>
+      <div class="alert alert-alert">
+        <%= show_errors(new_post, :base) %>
+      </div>
+    <% end %>
+    <div class="tabbed">
+      <%= post.label :title %>
+      <%= post.text_field :title %><br/>
+    </div>
+    <% if new_post.errors[:title].present? && !new_post.errors[:base].present? %>
```

Comment:
why the check for `:base` errors here? Is the idea to not show inline errors if there are base errors?

Reply:
Exactly!